packages/desktop-client/src/components/common/Search.tsx

```diff
@@ -30,13 +30,8 @@ export default function Search({
       style={{
         width,
         borderColor: isInModal ? null : 'transparent',
-        backgroundColor: isInModal ? null : colors.n11,
-        ':focus': isInModal
-          ? null
-          : {
-              backgroundColor: 'white',
-              '::placeholder': { color: colors.n8 },
-            },
+        backgroundColor: isInModal ? null : theme.formInputBackground,
+        color: 'inherit',
```

Comment:
Is this override necessary?

Reply:
This was there for testing, thanks for catching it. Removed.